### ISSUE TITLE
Completely broken in Firefox 87.0

### ISSUE THREAD

**Issue by sebmaynard** [excalidraw/excalidraw#3262]
I'm a long-time, multiple-times-per-day Excalidraw user, and today it no longer works in Firefox on Windows. Seems fine on Chrome.

The console shows 2 warnings:

    The resource at “https://excalidraw.com/Virgil.woff2” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly. excalidraw.com

    The resource at “https://excalidraw.com/Cascadia.woff2” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.

I can pick a tool, but trying to draw anything produces no visible change to the canvas, and then immediately selects the arrow tool again.

**Comment by dwelle** [comment]
FF 87, is that beta? I can't reproduce either on 86 stable not 87.0b9 beta, Windows.

What does it say when you paste `__EXCALIDRAW_SHA__` into the console? If `84a1863233753294b6254d444a8d04e1780a2585`, then you're on latest.

**Comment by dwelle** [comment]
What you're describing could be caused by an invalid state. Can you try exporting the scene if it's not private?

If export (save as) doesn't work either, you can paste this into the console (it'll copy the contents to your clipboard), and post it here.

```
copy(({...localStorage}))
```

**Comment by sebmaynard** [comment]
Good morning! I tried in Private Browsing mode and got the same behaviour.

I just restarted Firefox in Safe Mode, everything worked as expected... restarted in "normal" mode and we're back in action.

Not entirely sure which of my add-ons was causing a problem (Privacy Badger, uBlock Origin) but seems to have been caused by one of them, will keep an eye out if it happens again, sorry for the noise!
